open_spiel/games/universal_poker.cc

```diff
@@ -1050,6 +1320,68 @@ open_spiel::Action ACPCActionToOpenSpielAction(
   return kInvalidAction;
 }
 
+std::shared_ptr<const Game> MakeRandomSubgame(std::mt19937& rng,
+                                              int pot_size,
+                                              std::string board_cards,
+                                              std::vector <int> hand_reach) {
+
+  constexpr const char* base_game =
+      "universal_poker("
+      "betting=nolimit,"
+      "numPlayers=2,"
+      "numRounds=4,"
+      "blind=100 50,"
+      "firstPlayer=2 1 1 1,"
+      "numSuits=4,"
+      "numRanks=13,"
+      "numHoleCards=2,"
+      "numBoardCards=0 3 1 1,"
+      "stack=20000 20000,"
+      "bettingAbstraction=fcpa,"
+      "potSize=%d,"
+      "boardCards=%s,"
+      "handReaches=%s"
+    ")";
+
+  if (pot_size == -1) {
+    // Multiples of 50, from 0 to 40000
+    // FIXME maybe there are other possible pot sizes?
+    std::uniform_int_distribution<int> dist(0, 800);
+    pot_size = dist(rng) * 50;
```

Comment:
maybe it's better to define 50 as variable, in case we want to try other values than 50, it's easier to find it and change the value.

Reply:
Now that I think of it these should match the HUNL game string, I updated the comment and range of bets.